.travis.yml

```diff
@@ -38,7 +38,6 @@ before_install:
     conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
     conda update conda --quiet
     conda config --add channels conda-forge --force
-    conda install pycryptosat
```

Comment:
@Conengmo this is unrelated to the problem here but I wanted to remove this b/c we are not really using it.